opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -346,7 +346,7 @@ class Span(trace_api.Span):
         name: The name of the operation this span represents
         context: The immutable span context
         parent: This span's parent's `opentelemetry.trace.SpanContext`, or
```

Comment:
Maybe comment should be `opentelemetry.context.Context` in which to obtain the current span.

Reply:
I'm not sure I follow, in this case the parent is a parent span context

Reply:
Oh sorry, this is in `Span`, so parent span context is still allowed. Forget what I said.